contracts/pol/interfaces/ICarbonPOL.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { IUpgradeable } from "../../utility/interfaces/IUpgradeable.sol";
+import { Token } from "../../token/Token.sol";
+
+/**
+ * @dev CarbonPOL interface
+ */
+interface ICarbonPOL is IUpgradeable {
+    error DuplicateToken();
+    error InvalidToken();
+    error InvalidTokenLength();
+    error TokenPoolFeesLengthMismatch();
+
+    /**
+     * @dev triggered after a successful trade is executed
+     */
+    event TokensTraded(
+        address indexed caller,
+        Token[] tokens,
+        uint24[] poolFees,
+        uint256[] tradeAmounts,
+        uint256[] rewardAmounts,
+        uint256 ethReceived
+    );
+
+    /**
+     * @dev triggered when the rewards ppm are updated
+     */
+    event RewardsUpdated(
+        uint32 prevRewardsPPM,
+        uint32 newRewardsPPM
+    );
+
+    /**
+     * @dev triggered when the max slippage ppm is updated
+     */
+    event MaxSlippageUpdated(
+        uint32 prevMaxSlippagePPM,
+        uint32 newMaxSlippagePPM
+    );
+
+    /**
+     * @dev triggered when the max tradeable ppm per block is updated
+     */
+    event MaxTradeableUpdated(
+        uint32 prevMaxTradeablePPB,
+        uint32 newMaxTradeablePPB
+    );
+
+    /**
+     * @dev returns the rewards ppm
+     */
+    function rewardsPPM() external view returns (uint32);
+
+    /**
+     * @dev returns the max slippage ppm
+     */
+    function maxSlippagePPM() external view returns (uint32);
+
+    /**
+     * @dev returns the max tradeable ppb per block
```

Comment:
It's per block. Set initially to 1389 points per billion, because we discussed having the max tradeable amount increasing by 1% of the token balance every 24 hours.
Since the merge, blocks are mined every 12 seconds precisely, so 5 per minute. That means, we have exactly 5 * 60 * 24 = 7,200 blocks every 24 hours. To get the max tradeable % increase per block, we need to divide 1% by 7,200 blocks = 0.000001389% per block.